src/components/sbb-menu/sbb-menu.tsx

```diff
@@ -336,7 +362,32 @@ export class SbbMenu implements ComponentInterface {
     this._element.style.setProperty('--sbb-menu-max-height', menuPosition.maxHeight);
   }
 
+  /**
+   * Create an array with only the sbb-menu-action children
+   */
+  private _readActions(): void {
+    const actions = Array.from(this._element.children);
+    // If the slotted actions have not changed, we can skip syncing and updating the actions.
+    if (
+      this._actions &&
+      actions.length === this._actions.length &&
+      this._actions.every((e, i) => actions[i] === e)
+    ) {
+      return;
+    }
+
+    if (actions.every((e) => e.tagName === 'SBB-MENU-ACTION')) {
+      this._actions = actions as HTMLSbbMenuActionElement[];
+    } else {
+      this._actions = undefined;
```

Comment:
In this case we might have to remove a previously assigned slot attribute, as otherwise an action could simply disappear
```suggestion
      this._actions?.forEach((a) => a.removeAttribute('slot'));
      this._actions = undefined;
```